datasets/mac_morpho/README.md

```diff
@@ -0,0 +1,172 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- pt
+licenses:
+- cc-by-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- part-of-speech-tagging
```

Comment:
```suggestion
- structure-prediction-other-part-of-speech-tagging
```
we've not added it yet to the taxonomy. 
I'll update this readme once it's done

Reply:
Done!